src/entries/popup/components/Menu/MenuItem.tsx

```diff
@@ -116,12 +118,14 @@ interface DescriptionProps {
   color?: TextStyles['color'];
   text: string;
   weight?: TextStyles['fontWeight'];
+  descriptionLink?: () => void;
```

Comment:
IMO this name should be `onClickLink` or something like that

as `descriptionLink` i understood is a string containing the link, not the action

Reply:
That makes sense to me. will change.